plugins/module_utils/attachment.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# Copyright: (c) 2021, XLAB Steampunk <steampunk@xlab.si>
+# Copyright: (c) 2021--2022, XLAB Steampunk <steampunk@xlab.si>
```

Comment:
```suggestion
# Copyright: (c) 2021, XLAB Steampunk <steampunk@xlab.si>
```

Keep this simple by only including the year in which the PR / code is submitted.